packages/ts/react-crud/src/header-filter.tsx

```diff
@@ -110,7 +93,7 @@ function ComparationSelection({ onMatcherChanged, value }: ComparationSelectionP
           </Item>
         </ListBox>
       )}
-      className={autoGridFilterWithLessGreaterEqualsStyle.id}
+      className="autoGridFilterWithLessGreaterEquals"
```

Comment:
Can this class be in kebab-case as well as all others?

Reply:
Done